unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -41,7 +41,7 @@ public enum State
         public ContentProvider contentProvider;
         public int disposableNotReadyCount => disposableNotReady.Count;
 
-        [System.NonSerialized] public bool useBlockers = true;
+        [System.NonSerialized] public bool useBlockers = false;
```

Comment:
Where is this param being used?

Reply:
for example here: https://github.com/decentraland/explorer/blob/d9dad36411b16b238f2928f1d23d06d76b2b8fb5/unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs#L143

That bool is to instantiate (or not) the scene blockers, since this PR puts blockers around the explorable area of the world (cenes that didn't finish loading don't count as explorable area), the scene blockers are not necessary anymore

Reply:
Oh, ok! I didn't see it in ParcelScene. Make sense. Thanks!